CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# CONTRIBUTING
+
+When contributing to this repository, please first discuss the change you wish to make via [issues](https://github.com/rigd-loxia/builder-generator/issues).
+
+Please note if you are working on a certain issue then make sure to stay active with development.
+
+## Git Commit, Branch, and PR Naming Conventions
+
+When you are working with git, please be sure to follow the conventions below on your pull requests, branches, and commits:
+
+```text
+PR: [#ISSUE ID] Title of the PR
```

Comment:
PR text should contain a line saying:
closing #[IssueId]
To link the issue that is closed by this PR correctly.

Reply:
changed PR to PR Title and PR description to make it more clear